packages/documentation-site/patternfly-docs/patternfly-docs.source.js

```diff
@@ -6,102 +6,88 @@ module.exports = (sourceMD, sourceProps, sourceFunctionDocs) => {
-    .resolve('@patternfly/react-charts/package.json')
-    .replace('package.json', 'src');
-
-  const reactLogViewerPath = require
-    .resolve('@patternfly/react-log-viewer/package.json')
-    .replace('package.json', 'src');
-  const reactTopologyPath = require
-   .resolve('@patternfly/react-topology/package.json')
-   .replace('package.json', 'patternfly-docs/content/examples');
-  const reactUserFeedbackPath = require
-   .resolve('@patternfly/react-user-feedback/package.json')
-   .replace('package.json', 'patternfly-docs/');
-
-  const logViewerContentBase = require
-    .resolve('@patternfly/react-log-viewer/package.json')
-    .replace('package.json', 'patternfly-docs/content/extensions/react-log-viewer');
-
   const reactPropsIgnore = ['**/*.test.tsx', '**/examples/*.tsx'];
-  sourceProps(path.join(reactCorePath, '/**/*.tsx'), reactPropsIgnore);
-  sourceProps(path.join(reactTablePath, '/**/*.tsx'), reactPropsIgnore);
-  sourceProps(path.join(reactCodeEditorPath, '/**/*.tsx'),reactPropsIgnore);
-  sourceProps(path.join(reactChartsPath, '/**/*.tsx'),reactPropsIgnore);
-  sourceProps(path.join(reactLogViewerPath, '/**/*.tsx'), reactPropsIgnore);
-  sourceProps(path.join(reactTopologyPath, '/**/*.tsx'), reactPropsIgnore);
-  sourceProps(path.join(reactUserFeedbackPath, '/**/*.tsx'), reactPropsIgnore);
-
-  // React MD
-  sourceMD(path.join(reactCorePath, '/components/**/examples/*.md'), 'react');
-  sourceMD(path.join(reactCorePath, '/layouts/**/examples/*.md'), 'react');
-  sourceMD(path.join(reactCorePath, '/next/components/**/examples/*.md'), 'react-next');
-  sourceMD(path.join(reactCorePath, '/deprecated/components/**/examples/*.md'), 'react-deprecated');
-  sourceMD(path.join(reactCorePath, '/**/demos/**/*.md'), 'react-demos');
-
-  // React-table MD
-  sourceMD(path.join(reactTablePath, '/components/**/examples/*.md'), 'react');
-  sourceMD(path.join(reactTablePath, '/deprecated/components/**/examples/*.md'), 'react-deprecated');
-  sourceMD(path.join(reactTablePath, '/**/demos/*.md'), 'react-demos');
-
-  // Charts MD (no demos yet)
-  sourceMD(path.join(reactChartsPath, '/**/examples/*.md'), 'react');
-
-  // React-code-editor MD
-  sourceMD(path.join(reactCodeEditorPath, '/**/examples/*.md'), 'react');
-
-  // React-log-viewer MD
-  sourceMD(path.join(logViewerContentBase, '/**/examples/*.md'), 'react');
-  sourceMD(path.join(logViewerContentBase, '/**/demos/*.md'), 'react-demos');
-
-  // React-topology MD
-  sourceMD(path.join(reactTopologyPath, '/**/*.md'), 'extensions');
-
-  // React-user-feedback MD
-  sourceMD(path.join(reactUserFeedbackPath, '/**/examples/*.md'), 'react');
-  sourceMD(path.join(reactUserFeedbackPath, '/**/design-guidelines/*.md'), 'design-guidelines');
-
-  // React OUIA MD
-  sourceMD(path.join(reactCorePath, '/**/helpers/OUIA/*.md'), 'react');
+  
+  if (process.env.EXTENSIONS_ONLY === 'true') { 
+    sourceMD(path.join(contentBase, 'extensions/**/*.md'), 'extensions');
```

Comment:
This line is called whether or not the build is extensions-only or regular, as both builds include the extension docs, so it's repeated below on [line 20](https://github.com/patternfly/patternfly-org/pull/3747/files#diff-4f2188cb353f2766c010a081762bb64556eb7bf758649c76cb09e4b79c29beb8R20).

To clean up a bit can we pull this extensions sourcing outside of the `if/else` statement, and then flip the logic to only run the rest of the `sourceMD` lines `if(!(process.env.EXTENSIONS_ONLY === 'true'))`?  To me that makes it a little clearer that the extension files are always sourced, and then the rest are sourced only if extensions_only isn't true.